cmd/main.go

```diff
@@ -257,6 +259,34 @@ func runDev(ctx context.Context, args []string) error {
 	return cmd.Wait()
 }
 
+func setcapIfRequested(output string) error {
+	if os.Getenv("XCADDY_SETCAP") != "1" {
+		return nil
+	}
+
+	args := []string{"setcap", "cap_net_bind_service=+ep", output}
+
+	// check if sudo isn't available, or we were instructed not to use it
+	_, sudoNotFound := exec.LookPath("sudo")
+	skipSudo := sudoNotFound != nil || os.Getenv("XCADDY_SETCAP_NO_SUDO") == "1"
```

Comment:
```suggestion
	skipSudo := sudoNotFound != nil || os.Getenv("XCADDY_SUDO") == "0"
```

Maybe something like this? Since if we don't want to run `sudo` for `setcap`, we probably don't want to run it for anything else (which we don't currently, but potentially in the future, I dunno -- I am just saying it's not `setcap`-specific).

And probably makes sense to just set it to "0" to disable it, since the default is empty string.